infrastructure/main.tf

```diff
@@ -32,4 +32,16 @@ module "case-management-web" {
     PRINT_SERVICE_URL_REMOTE = "${var.ccd_print_service_url}"
     WEBSITE_NODE_DEFAULT_VERSION = "8.9.4"
   }
+
+  module "ccd-case-management-web-vault" {
+    source              = "git@github.com:hmcts/moj-module-key-vault?ref=master"
+    name                = "ccd-case-management-web${var.env}" // Max 24 characters
+    product             = "${var.product}"
+    env                 = "${var.env}"
+    tenant_id           = "${var.tenant_id}"
+    object_id           = "${var.jenkins_AAD_objectId}"
+    resource_group_name = "${module.ccd-case-management-web.resource_group_name}"
```

Comment:
This should be a reference to https://github.com/hmcts/ccd-case-management-web/pull/29/files#diff-1031eeab417cd8afe77e7e005717d38dR9 so:
```
resource_group_name = "${module.case-management-web.resource_group_name}"
```